snaps/features/custom-evm-accounts/security.md

```diff
@@ -28,20 +28,20 @@ For example:
 
     ```ts
     const account: KeyringAccount = {
-      id: uuid(),
-      options: {
-        privateKey: '0x01234...78', // !!! DO NOT DO THIS !!!
-      },
-      address,
-      methods: [
-        EthMethod.PersonalSign,
-        EthMethod.Sign,
-        EthMethod.SignTransaction,
-        EthMethod.SignTypedDataV1,
-        EthMethod.SignTypedDataV3,
-        EthMethod.SignTypedDataV4,
-      ],
-      type: EthAccountType.Eoa,
+        id: uuid(),
+        options: {
+            privateKey: "0x01234...78", // !!! DO NOT DO THIS !!!
```

Comment:
Maybe more descriptive comment in addition to the  "DO NOT DO THIS"

Reply:
The text preceding the code sample provides context on what not to do, I think this comment is just highlighting the line where it applies: https://docs.metamask.io/snaps/features/custom-evm-accounts/security/#do-not-add-secret-information-to-account-objects